examples/pytorch/speech-recognition/run_speech_recognition_ctc.py

```diff
@@ -132,6 +134,12 @@ class ModelArguments:
     ctc_loss_reduction: Optional[str] = field(
         default="mean", metadata={"help": "The way the ctc loss should be reduced. Should be one of 'mean' or 'sum'."}
     )
+    adapter_attn_dim: int = field(
+        default=None,
```

Comment:
Can this default to some sensible value or should we always force the user to pass it?

Reply:
Yes good point, now that things will be moved to a new file, I'll set a good default